docs/en-US/component/pagination.md

```diff
@@ -80,9 +80,9 @@ pagination/more-elements
 | page-sizes                          | options of item count per page                                                                                                  | ^[object]`number[]`                                                               | [10, 20, 30, 40, 50, 100]            |
 | popper-class                        | custom class name for the page size Select's dropdown                                                                           | ^[string]                                                                         | ''                                   |
 | prev-text                           | text for the prev button                                                                                                        | ^[string]                                                                         | ''                                   |
-| prev-icon                           | icon for the prev button, higher priority of `prev-text`                                                                        | ^[string] / ^[Component]                                                          | ArrowLeft                            |
+| prev-icon                           | icon for the prev button, lower priority of `prev-text`                                                                         | ^[string] / ^[Component]                                                          | ArrowLeft                            |
```

Comment:
I think the current expression is not very clear. I would suggest: **_has a lower priority than \`prev-text\`_**. The same below.